tests/foreman/api/test_rhcloud_inventory.py

```diff
@@ -406,3 +407,48 @@ def test_rh_cloud_tag_values(
                 if tag['key'] == 'host_collection':
                     assert tag['value'] == f'"{host_col_name}"'
                     break
+
+
+@pytest.mark.run_in_one_thread
+@pytest.mark.tier2
+def test_positive_tag_values_max_length(
+    inventory_settings, organization_ak_setup, rhcloud_registered_hosts, rhcloud_sat_host
+):
+    """Verify that tags values are truncated properly for the host parameter
+            with max length.
+
+    :id: dbcc7245-88af-4c35-87b8-92de01030cb5
+
+    :Steps:
+        1. Enable include_parameter_tags setting
+        2. Create a host parameter with long text value.
+        3. Generate an rh_cloud reportq
+        4. Observe the tag generated from the parameter.
+
+    :expectedresults:
+        1. Parameter tag value must must not be created after the
+                allowed length.
```

Comment:
One extra `must` word here. And I suggest to format it differently if there is only single expected result.
```suggestion
    :expectedresults: Parameter tag value must not be created after the allowed length.
```